src/main/java/com/mathworks/ci/Utilities.java

```diff
@@ -45,8 +45,9 @@ public static void addMatlabToEnvPathFrmAxis(Computer cmp, TaskListener listener
         }
 
         String matlabExecutablePath = getNodeSpecificHome(name,
-               cmp.getNode(), listener, env) + "/bin";
+               cmp.getNode(), listener, env) + ((Boolean.TRUE.equals(cmp.isUnix()))?"/bin" : "\\bin");
```

Comment:
Why do we need Boolean.TRUE.equal ? does't just cmp.isUnix() ? a : b works

Reply:
isUnix() has @ CheckForNull annotation and may return null pointer exception, This is a fail-safe